### ISSUE TITLE
Changing floppy images with CTRL + F4 sends Dosbox to the background in macOS when other windows are open

### ISSUE THREAD

**Issue by Burrito78** [dosbox-staging/dosbox-staging#992]
Hi,

i mount a few floppy images with:

`imgmount a test1.img test2.img test3.img -t floppy`

When trying to switch between these image files with `CTRL` + `F4`, the Dosbox window gets always minimized and the switching itself doesn't always work correctly (because of this?).

The option in macOS to use F1-F12 as standard function keys is activated.

Regards,
Burrito78

**Comment by kcgen** [comment]
Thanks @Burrito78 - we certainly don't want conflicting hotkeys if `Ctrl + F4` is a common macOS system shortcut.

I'm looking https://support.apple.com/en-ca/HT201236, but can't find `F4` in the list; but I might not know what to look for (if that combination is given another name).

Can you help point it out?

**Comment by Burrito78** [comment]
This happens on a Macbook Pro with latest macOS (11.3) and a Mini running Catalina (10.15.7). Magnet app isn't open to prevent non-standard window behavior.

The problem doesn't happen when Dosbox is the only app on the desktop. Also imgmount switching works flawlessly then.

It only happens when at least one background window is present. 

What `ctrl + F4` does is "bring the last used app to the foreground". Therefore Dosbox does get pushed to the background (and not minimized like i wrote initially). I can't find any documentation of this shortcut either.

Regards,
Alex

**Comment by Burrito78** [comment]
Ok, found it. It's called "Move focus to active or next window". 

https://apple.stackexchange.com/a/29185

**Comment by kcgen** [comment]
Thank you @Burrito78 - this sounds like a show-stopper.. :man_facepalming: 

Adding @grounded0 and @kklobe.

Is there a different modifier key that would overwhelmingly better (and not conflict) on macOS?

Would it be true to say that macOS users simply steer-clear of these hotkeys in DOSBox to avoid performing various window actions? 

If this is the case, then muscle-memory doesn't exist amung the macOS user community - and we have only upside to gain to switching to a non-conflicting modifier (on macOS-only).

**Comment by kklobe** [comment]
I did some testing here and it looks like a quick work around is to use Ctrl+F1 to disable keyboard access, which seems to be on by default (I tested by creating a new user and logging in). After that, I was able to verify that Ctrl+F4 executed the proper DOSBox command instead of "move focus".

Long-term it might be best to add a modifier key to those Ctrl+F? mappings, maybe Option or Command - it doesn't look like either would conflict with defaults, after looking at https://support.apple.com/en-us/HT201236 and http://www.keyxl.com/aaa1f76/48/Apple-Mac-OS-X-OSX-keyboard-shortcuts.htm.
